apps/web/src/pages/ApplicantDashboardPage/components/ReviewApplicationDialog.tsx

```diff
@@ -0,0 +1,429 @@
+    removedAt
+    finalDecision {
+      value
+    }
+    assessmentStatus {
+      assessmentStepStatuses {
+        step
+      }
+      overallAssessmentStatus
+      currentStep
+    }
+    pool {
+      id
+      name {
+        localized
+      }
+      classification {
+        group
+        level
+        minSalary
+        maxSalary
+      }
+      workStream {
+        name {
+          localized
+        }
+      }
+      language {
+        value
+        label {
+          localized
+        }
+      }
+      department {
+        name {
+          localized
+        }
+      }
+      closingDate
+      isRemote
+      location {
+        localized
+      }
+      securityClearance {
+        value
+        label {
+          localized
+        }
+      }
+      areaOfSelection {
+        value
+      }
+      processNumber
+      poolSkills {
+        id
+        type {
+          value
+          label {
+            localized
+          }
+        }
+        skill {
+          id
+          key
+          name {
+            localized
+          }
+          category {
+            value
+            label {
+              localized
+            }
+          }
+        }
+      }
+      screeningQuestions {
+        id
+      }
+    }
+  }
+`);
+
+interface ReviewApplicationDialogProps {
+  applicationQuery: FragmentType<typeof ReviewApplicationDialog_Fragment>;
+}
+
+const ReviewApplicationDialog = ({
+  applicationQuery,
+}: ReviewApplicationDialogProps) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const paths = useRoutes();
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const application = getFragment(
+    ReviewApplicationDialog_Fragment,
+    applicationQuery,
+  );
+
+  const pool = application?.pool;
+
+  const nullMessage = intl.formatMessage(commonMessages.notFound);
+  const poolName =
+    pool.name?.localized ?? intl.formatMessage(commonMessages.notFound);
+
+  // Separate essential and asset skills, sort them by category, and confirm they include skill data
+  const poolSkills = unpackMaybes(pool?.poolSkills);
+  const essentialPoolSkills = sortPoolSkillsBySkillCategory(
+    poolSkills.filter(
+      (poolSkill) => poolSkill.type?.value === PoolSkillType.Essential,
+    ),
+  );
+  const nonessentialPoolSkills = sortPoolSkillsBySkillCategory(
+    poolSkills.filter(
+      (poolSkill) => poolSkill.type?.value === PoolSkillType.Nonessential,
+    ),
+  );
+
+  const focusOnRecruitment = useRef(false);
+
+  const status = getApplicationStatusChip(
+    application.submittedAt,
+    pool.closingDate,
+    application.removedAt,
+    application.finalDecisionAt,
+    application.finalDecision?.value,
+    pool.areaOfSelection?.value,
+    application.assessmentStatus,
+    pool.screeningQuestions,
+    intl,
+  );
+
+  return (
+    <Dialog.Root open={isOpen} onOpenChange={setIsOpen}>
+      <Dialog.Trigger asChild>
+        <PreviewList.Button label={poolName} />
```

Comment:
We got the title for the card updated with the additional context. However, we need to replicate that here and possibly toss an action word in. The trigger for the dialog is just as important as the heading to distinguish it from the other items on the page.

Reply:
Added in here [e9fa102](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/e9fa10262e89daca332eefdd9ec54cc02242e5e8)